src/main/java/com/opsmatters/newrelic/api/model/graphql/NrqlSuccessResponse.java

```diff
@@ -0,0 +1,67 @@
+package com.opsmatters.newrelic.api.model.graphql;
+
+import com.google.gson.JsonArray;
+
+import java.util.List;
+
+/**
+ * Represents Graph QL response data.
```

Comment:
nit: I've noticed this a few times in this repo, but it's typically stylized as one word
```suggestion
 * Represents GraphQL response data.
```